toolchest_client/cli/cli.py

```diff
@@ -0,0 +1,17 @@
+import sentry_sdk
```

Comment:
The Sentry SDK is not longer in the project, this + the tag below should be removed

Reply:
@jherr-dev before I continuing reviewing, was this tested? Could have passed if you have `sentry_sdk` installed elsewhere